Src/Support/GoogleApis.Auth.NetStandard/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
I think it would be good to trim these dependencies - see https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/reducing-dependencies